cmd/state/internal/cmdtree/show.go

```diff
@@ -18,7 +18,14 @@ func newShowCommand(prime *primer.Values) *captain.Command {
 		locale.T("show_project"),
 		prime.Output(),
 		prime.Config(),
-		nil,
+		[]*captain.Flag{
+			{
+				Name:        "detail",
+				Shorthand:   "d",
+				Description: locale.T("arg_state_show_remote_description"),
+				Value:       &params.Detail,
+			},
+		},
```

Comment:
I don't think this warrants a flag. Including the project path seems like something sensible that we should just always do.

Reply:
That makes it easier. I'll remove all the flag bits then.